utils/build_appimage.sh

```diff
@@ -17,7 +17,15 @@ mv AppDir/usr/share/games/endless-sky/* AppDir/
 
 # Now build the actual AppImage
 curl -sSL https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage -o linuxdeploy && chmod +x linuxdeploy
-./linuxdeploy --appdir AppDir -e endless-sky -d endless-sky.desktop -i endless-sky.png --output appimage
+OUTPUT=Endless_Sky-continuous-x86_64.AppImage ./linuxdeploy --appdir AppDir -e endless-sky -d endless-sky.desktop -i endless-sky.png --output appimage
+
+# Use the static runtime for the AppImage. This lets the AppImage being run on systems without fuse2.
+APPTOOL=$(curl -sSL https://github.com/probonopd/go-appimage/releases/expanded_assets/continuous | grep "appimagetool-.*-x86_64.AppImage" | head -n 1 | cut -d '"' -f 2)
+curl -sSL https://github.com$APPTOOL -o appimagetool && chmod +x appimagetool
+
+./Endless_Sky-continuous-x86_64.AppImage --appimage-extract
+chmod -R 755 ./squashfs-root
+./appimagetool ./squashfs-root
```

Comment:
This is hideous, but I guess it's our only option until linuxdeploy catches up

Reply:
Until the static runtime is officially blessed yeah unfortunately